### ISSUE TITLE
web support broken and does not seem to be able to log out even using sessionlogout

### ISSUE THREAD

**Issue by telostia** [heroiclabs/nakama-dart#105]
version: latest 1.1.0
flutter version: 3.19.6
dart version: 3.3.4
platform: web
problem: flutter app cannot relog in due to session not able to logged out. 
tried solution: 1)tried sessionLogout 2) setting session to null 3) setting client to null 4) setting websocket to null.
result: the nakama server session seems to linger with all the methods tried above
force solution: force refresh the web app will remove server session status.

Is there an more elegant solution to this?

**Comment by telostia** [comment]
this is the code i used : 
`Future<void> login(String username, String password) async {
    var session = await NakamaClient.getInstance()!
        .authenticateEmail(email: username, password: password, create: false);

    emit(ApplicationState(session: session, match: state.match));
    NakamaWSClient.setSession(session);
  }

  void logout(Session session) {
    try {
      var client = NakamaClient.getInstance();
      client?.sessionLogout(session: session);
      client = null;
    } catch (onError) {
      print(onError.toString());
    }
    // NakamaWSClient.setSession(session);
  }`

**Comment by telostia** [comment]
found the culprit... it was the appbar widget that was causing connections to go berserk. somehow placing the login/logout functions here was causing nakama sockets to now show up on the server debug...

**Comment by telostia** [comment]
Nope. the issue was due to " await getAccount(session)"... causing an issue for dart client not to relogin.

cause: calling  `await getAccount(session)`
Error: {"code":16,"message":"Server key invalid"}

**Comment by ilmalte** [comment]
Hey @telostia thanks for reporting this issue.
Many improvements and bug fixes have been pushed to `main` since the version 1.1.0, and they will be released soon.
You might want to try to update your `pubspec.yaml` file to point directly to `main` for now:

```
dependencies:
  nakama:
    git:
      url: 'https://github.com/heroiclabs/nakama-dart.git'
      path: nakama
```
I am not certain if this will resolve your issue, so I will attempt to reproduce it on my machine and will keep you updated.

**Comment by ilmalte** [comment]
Possible related issue to #108, potentially addressed by #110.

**Comment by ilmalte** [comment]
I'll close this issue as version v1.2.0 has been published, which should resolve it. If the problem persists, please feel free to reopen the issue with additional context. Thank you!
